pkgs/development/go-modules/generic/default.nix

```diff
@@ -19,6 +19,9 @@
 # path to go.mod and go.sum directory
 , modRoot ? "./"
 
+# Flags forwarded to the `go mod` call site.
+, goModFlags ? []
```

Comment:
Don't need to introduce another specific flag, use `NIX_DEBUG` which is intended for this and is already used here.

https://github.com/NixOS/nixpkgs/blob/32c523914fdb8bf9cc7912b1eba023a8daaae2e8/pkgs/development/go-modules/generic/default.nix#L208-L210

Reply:
thanks - updated